bundles/net.tourbook.chart/src/net/tourbook/chart/ChartComponentGraph.java

```diff
@@ -2970,8 +2934,11 @@ private void drawAsync_510_LineGraphSegment(final GC gc,
                      path.lineTo((int) devXPrevNoLine, (int) (devY0Inverse - devY1Prev));
                   }
 
-                  devY = devY0Inverse - devY1;
-
+                  if (yData.isYAxisDirection()) {
```

Comment:
@wolfgang-ch This is the 2nd part of the fix.

The rest is code improvements I found when I modified those files.